src/api/types.ts

```diff
@@ -0,0 +1,49 @@
+export interface Exercise {
+    name: string,
+    img: string,
+    muscles?: string[],
+    sets: string,
+    reps: string,
+    weight: string,
+    unit: "kg" | "lb",
+    rest: string
+}
+
+export interface Workout {
+    name: string,
+    exercises: Exercise[],
+    id?: number
+}
+
+export interface WholeWorkout {
+    created_at: string,
+    id: number,
+    user_id: string,
+    workout: Workout
+}
+
+interface WorkoutError {
+    code: string
+    details: null
+    hint: null
+    message: string
+}
```

Comment:
Isn't it a response error?

Reply:
Yeah, I renamed it yestarday while refactoring some other component.